inc/compatibility/woocommerce/class-astra-woocommerce.php

```diff
@@ -1665,7 +1710,7 @@ public function add_styles() {
 				),
 			);
 			/* Parse CSS from array()*/
-			$yith_wcwl_main_style .= astra_parse_css( $yith_wcwl_main_style_small, '', astra_get_tablet_breakpoint() );
+			$yith_wcwl_main_style .= astra_parse_css( $yith_wcwl_main_style_small, astra_get_tablet_breakpoint() );
```

Comment:
```suggestion
			$yith_wcwl_main_style .= astra_parse_css( $yith_wcwl_main_style_small, '', astra_get_tablet_breakpoint() );
```
@GauravKhupse Any reason for this change?

Reply:
This is showing code analysis changes but now I have fixed using ignore the rule.